hr_holidays_public/models/resource_calendar.py

```diff
@@ -42,12 +42,20 @@ def _public_holidays_leave_intervals(self, start_dt, end_dt, employee_id, tz):
             lines = HrHolidaysPublic.get_holidays_list(
                 day.year, employee_id=employee_id
             )
+            # In some cases, an error appears about mixing 2 models
+            # (hr.holidays.public.line + resource.calendar.leaves)
+            # in _leave_intervals_batch or _leave_intervals_batch functions
```

Comment:
I'm seeing now that it's the same function twice. Do you mean another?

Reply:
It's correct, 2 different functions are needed.
In 14.0 `_leave_intervals` remove, https://github.com/OCA/hr-holidays/blob/13.0/hr_holidays_public/models/resource_calendar.py#L56

Reply:
Then you should put:
```suggestion
            # in _leave_intervals or _leave_intervals_batch functions
```